### PR TITLE
Create OpenFoodFacts-Android.yml

### PR DESCRIPTION
**Pull request by teolemon**
Added an entry for the Open Food Facts Android app. We have a web version, iOS client, and many other interesting projects as well that I will add in other PRs if there is interest.

### CODE REVIEW

File: _data/projects/OpenFoodFacts-Android.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+desc: Open Food Facts is the Wikipedia for Food. An open-source & open-data project, it allows you to inform yourself and transform the food system by promoting its transparency, thanks to crowdsourcing via mobile apps
+site: https://github.com/openfoodfacts/openfoodfacts-androidapp
+tags:
+  - java
+  - food
+  - android
+  - nutrition
+  - environment
+upforgrabs:
+  name: up for grabs
```
Comment (shiftkey) [inline _data/projects/OpenFoodFacts-Android.yml]: This label should match a label used in your project.

File: _data/projects/OpenFoodFacts-Android.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  link: https://github.com/openfoodfacts/openfoodfacts-androidapp/labels/hacktoberfest
```
Comment (shiftkey) [inline _data/projects/OpenFoodFacts-Android.yml:12]: This URL should use the label above

**Review by shiftkey — COMMENTED** [review]
Thanks for submitting @teolemon!

A couple of changes need to be made based on what the automated review detected...

File: _data/projects/OpenFoodFacts-Android.yml (hunk 2 above)
Comment (teolemon) [reply _data/projects/OpenFoodFacts-Android.yml:12]: corrected

**Comment by teolemon** [comment]
Thank you @shiftbot ;-)

**Review by shiftkey — APPROVED** [review]
Thanks for the contribution @teolemon!

**Pull request merged**